src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,6 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+       return (UserService.findByEmail(email).getPassword() == password) ? true : false;
```

Comment:
make sure that User is not null before you call `.getPassword()`

compare strings the right way

and you can get simplify the ternary operator here as well